tests/end-to-end/director/runtime_api_test.go

```diff
@@ -66,22 +66,6 @@ func TestRuntimeCreateUpdateAndDelete(t *testing.T) {
 			}`, actualRuntime.ID, tc.gqlFieldsProvider.ForRuntime()))
 	err = tc.RunQuery(ctx, getRuntimeReq, &actualRuntime)
 	require.NoError(t, err)
-	assert.Len(t, actualRuntime.Labels, 2)
```

Comment:
maybe we should introduce feature flag that disables right now this part of test?

Reply:
It is removed because of lack of the functionality, right? I would be better to comment it then and add TODO to uncomment.